go.mod

```diff
@@ -16,6 +16,7 @@ require (
 	github.com/google/wire v0.4.0
 	github.com/googleapis/gax-go/v2 v2.0.5
 	github.com/kelseyhightower/envconfig v1.4.0
+	github.com/rickb777/date v1.13.0
```

Comment:
Is this library also the one used by Eventing for parsing these durations?

Reply:
Yes:
- https://github.com/knative/eventing/blob/master/go.mod#L23
- https://github.com/knative/eventing/blob/master/pkg/apis/duck/v1/delivery_types.go#L72